unlock-app/src/components/interface/modal-templates/WalletCheck.tsx

```diff
@@ -5,7 +5,7 @@ import { Dispatch } from '../../../unlockTypes' // eslint-disable-line
 
 const WalletCheck = ({ dispatch }: { dispatch: Dispatch }) => (
   <MessageBox>
-    <p>Please check your browser wallet to complete the transaction.</p>
+    <p>Please check your browser wallet.</p>
```

Comment:
Do you think it would be worth making this component take a property including the reason to check the wallet? That would let us have specific messages for transactions, signatures, and enabling access.

Reply:
Maybe for a later PR when the need arises?